wfw/lib.py

```diff
@@ -20,11 +22,16 @@
 TREE_DATA = expanduser('~/.wfwtree')
 USER_DATA = expanduser('~/.wfwrc')
 ROOT = {'id' : 0, 'text' : 'My list', 'children' : [], 'done' : False}
+aux = True
 
 
 def build_tree_from_file():
+    global ROOT
     with open(TREE_DATA, 'r') as tree_data:
-        build(tree_data, ROOT)
+        mjson = json.load(tree_data)
+        if aux:
```

Comment:
This is always going to be true, as `aux` is defined outside the function as true. I think`aux` should be passed to this function from `add_item`.